linera-service/tests/linera_net_tests.rs

```diff
@@ -2505,7 +2546,10 @@ async fn test_wasm_end_to_end_non_fungible(config: impl LineraNetConfig) -> Resu
     )
     .await;
 
-    assert_eq!(node_service2.process_inbox(&chain2).await?.len(), 1);
+    assert!(
+        eventually(|| async { node_service2.process_inbox(&chain2).await.unwrap().len() == 1 })
+            .await
+    );
```

Comment:
Those repetitions are annoying.

Reply:
I created https://github.com/linera-io/linera-protocol/issues/4718 to address this, but it will be a lot of work.